docs/api.rst

```diff
@@ -767,11 +767,10 @@ any consumers before it is terminated.
                          defaults to "binary".
    :<json string auto.offset.reset: Sets the ``auto.offset.reset`` setting for the consumer
    :<json string auto.commit.enable: Sets the ``auto.commit.enable`` setting for the consumer
-   :<json string fetch.min.bytes: Sets the ``fetch.min.bytes``
-                                                    setting for this consumer specifically
+   :<json string fetch.min.bytes: Minimal amount of bytes to return before timeout has been reached. This setting allows consumer to return before timeout is reached. Default is `-1`
```

Comment:
It is probably useful to define the setting here as well, but it is worth it to mention that this config is already documented in https://docs.confluent.io/current/kafka-rest/docs/config.html

I think that at the very least we should contain 
```
Sets the ``fetch.min.bytes`` proxy setting for this consumer specifically
```

Reply:
No, what I meant was that the 
```
Sets the ``fetch.min.bytes`` proxy setting for this consumer specifically
```
is sufficient documentation, because `fetch.min.bytes` is explained here -> https://docs.confluent.io/current/kafka-rest/docs/config.html

I think that your addition of
```
If you want to return earlier than that if a given byte threshold is reached, configure `fetch.min.bytes`.
```
should be in the `docs/config.rst` part of `consumer.request.timeout.ms`.

`docs/config.rst` is the authoritative source of truth for a configuration's behavior. I don't know if it's a good idea to duplicate that explanation in the `api.rst` docs

Reply:
@mente I believe you might have missed my point here.
The changes you made currently duplicate our definition of `fetch.min.byte`, which I think is unnecessary and will create extra burden when updating the config. Could you re-read my previous comment and share your thoughts?

Reply:
Point is that you need to connect the dots between the server configuration and REST api. Because in the big companies (IMHO where kafka is common solution) you might have different people setting up the proxy (admins, devops, bla-bla) and using the APIs of the server (developers, data engineers, etc.)